src/components/menus/definitions/InspectorDefinitionMenu.tsx

```diff
@@ -75,12 +75,12 @@ const InspectorDefinitionMenu = (props: InspectorDefinitionProps) => {
             // TODO: define error type
             const errors: any = {};
             const definition = definitions[dataMapping.type];
-            
-            const names = definition.map((d) => d.name);
+
+            const names = definition.map((d) => d.name.trim());
             const isNameDuplicate = names.includes(values.name);
```

Comment:
We should also trim the `values.name` to avoid unexpected results in the comparison should the user leave empty spaces in the definition name.

This raises the question of whether users can save definition names with empty spaces in the first place. If not, we don't need the trim at line 79.